README.md

```diff
@@ -79,6 +79,15 @@ site you're working on.
 For example, the [manifest for the docs section](https://github.com/cert-manager/website/blob/master/content/docs/manifest.json)
 contains the expected path for every file.
 
+### Task: Changing OpenGraph / social sharing tags
```

Comment:
I don't really know what OpenGraph and these tags are. However, I can google that if needed, else may also be valuable to ad some comments to the javascript in ./pages (not in this PR)

Reply:
Yeah fair enough really! I've got a task to do more on this in #934 so I might add more docs there!